### PR TITLE
Added note regarding Windows auth access denied.

### LINKED ISSUE #5494: Windows Authentication - Configure AccessDeniedPath
When an authorization policy fails a completely blank page is returned when using windows auth.  With other authentication options, like cookie authentication, one can configure an AccessDeniedPath to redirect a user too in this scenario.  What is the equivalent for windows authentication?   The [documentation](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?tabs=aspnetcore2x) seems to gloss over this detail.

### PR DESCRIPTION
**Pull request by Trcx528**
Fixes #5494

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?branch=pr-en-us-5512&tabs=aspnetcore2x#work-with-windows-authentication)

### CODE REVIEW

**Comment by scottaddie** [comment]
@Tratcher Please review.

**Comment by scottaddie** [comment]
@Trcx528 Thank you for adding this detail. We very much appreciate the time you took to open the issue and address it with this PR!

File: aspnetcore/security/authentication/windowsauth.md [hunk 1]
```diff
@@ -107,6 +107,9 @@ When both Windows authentication and anonymous access are enabled, use the `[Aut
 
 In ASP.NET Core 2.x, the `[Authorize]` attribute requires additional configuration in *Startup.cs* to challenge anonymous requests for Windows authentication. The recommended configuration varies slightly based on the web server being used.
 
+> [!NOTE]
+> By default, users who lack authorization to access a page are presented with a blank document. The [StatusCodePages middleware](xref:fundamentals/error-handling#configuring-status-code-pages) can be configured to provide users with a better "Access Denied" experience.
```
Comment (Tratcher) [inline aspnetcore/security/authentication/windowsauth.md]: "a blank document" -> "an empty 403 response"

Comment (scottaddie) [reply aspnetcore/security/authentication/windowsauth.md]: Thanks, Chris! I'll address this in https://github.com/aspnet/Docs/issues/5563.

**Pull request merged**